<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPersonDialog</class>
 <widget class="QWidget" name="AddPersonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(243, 243, 243);border:0.5px groove gray;border-radius:7px;padding:2px 4px;border-style: outset;}
QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}
QPushButton:pressed{background-color:rgb(191, 190, 192);border-style: inset;}</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgb(243, 243, 243);border:0.5px groove gray;border-radius:7px;padding:2px 4px;border-style: outset;}
QPushButton:hover{background-color:rgb(229, 241, 251); color: black;}
QPushButton:pressed{background-color:rgb(191, 190, 192);border-style: inset;}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>265</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>12</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>姓名</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="leftHorizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="nameLabel"/>
    </item>
    <item>
     <spacer name="rightHorizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>15</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>部门</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="depNameComboBox">
      <item>
       <property name="text">
        <string>办领导</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>综合处</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>政策法规处</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>机构改革处</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>党群政法行政机构编制管理处</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>政府行政机构编制管理处</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>市县行政机构编制管理处</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>事业机构编制管理处</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>事业单位登记管理处</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>机构编制监督检查处</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>人事处</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>机关党委</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>省机构编制电子政务中心</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>省机构编制研究中心</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
